tests/test__mongomock.py

```diff
@@ -735,6 +735,18 @@ def test__find_limit(self):
         # pymongo limit defaults to 0, returning everything
         self.cmp.compare.find(limit=0, sort=[("a", 1), ("b", -1)])
 
+    def test__find_projection_subdocument_lists(self):
+        self.cmp.do.remove()
+        self.cmp.do.insert({'a': 1, 'b': [{'c': 3, 'd': 4}, {'c': 5, 'd': 6}]})
+        for project in ({'_id': 0, 'a': 1, 'b': 1},
```

Comment:
It's not much more work to write `self.cmp.compare.find_one({'a': 1}, ` before each projection case, and it makes potential error easier to read (otherwise, one does not know which projection broke the test).